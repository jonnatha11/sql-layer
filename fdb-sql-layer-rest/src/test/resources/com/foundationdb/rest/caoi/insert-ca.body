[{
    "cid": 6,
    "first_name": "John",
    "last_name": "Smith",
    "test.addresses": [
        {
            "aid": 104,
            "cid": 6,
            "state": "MA",
            "city": "Boston"
        }
    ]
}]