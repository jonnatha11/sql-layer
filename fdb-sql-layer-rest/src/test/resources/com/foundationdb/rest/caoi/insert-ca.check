/entity/test.customers/6
