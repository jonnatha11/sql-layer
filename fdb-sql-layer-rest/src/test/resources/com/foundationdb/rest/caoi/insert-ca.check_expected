[{
    "cid": 6,
    "first_name": "John",
    "last_name": "Smith",
    "addresses": [
        {
            "aid": 104,
            "cid": 6,
            "state": "MA",
            "city": "Boston"
        }
    ]
}]