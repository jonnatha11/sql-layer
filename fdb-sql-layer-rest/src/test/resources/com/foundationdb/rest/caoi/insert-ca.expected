[{"cid": 6, "addresses": [{"aid": 104}]}]